mmv1/products/compute/api.yaml

```diff
@@ -8396,8 +8396,16 @@ objects:
         name: 'size'
         description: |
           The total number of nodes in the node group.
-        required: true
```

Comment:
It would be nice to keep this `required: true` in Ansible using `ansible.yaml`, non-blocking though.

Reply:
done